### ISSUE TITLE
Bug in Configs

### ISSUE THREAD

**Issue by MrHoller** [PowerNukkitX/PowerNukkitX#1192]
# 🐞 I found a bug

If you create two different plugins, write in onEnable() in both
`saveDefaultConfig()` and put different config.yml in resources of both plugins
Then when you turn on the server in both plugins, the same config from the first loaded plugin will be saved
sorry for my bad english

### 📸 Screenshots / Videos

![1](https://github.com/PowerNukkitX/PowerNukkitX/assets/35010313/de4aa325-65a8-430b-bfbc-4eb8610d4c52)
![2](https://github.com/PowerNukkitX/PowerNukkitX/assets/35010313/7a603b02-18bd-45bb-9f0f-596307f31030)

### ▶ Steps to Reproduce

1. Paste the two plugins into the plugins folder (plugins below)
2. Run server
3. Look config.yml

### ✔ Expected Behavior

Configs work fine

### ❌ Actual Behavior

Configs are not working correctly

### 📋 Debug information

* PowerNukkit version: 1.19.80-r2

### 💢 Crash Dump, Stack Trace and Other Files

[sources_and_jars.zip](https://github.com/PowerNukkitX/PowerNukkitX/files/11548379/sources_and_jars.zip)

**Comment by IndianBartonka** [comment]
i think it's not a bug, you just have two plugins with the same packgate names, next time name them like this for example:
``me.author name.PluginName``
``me.indian.Ostag``
You just need to have different packgate names :)

**Comment by IndianBartonka** [comment]
And you can also load config in `onLoad` method
```java
@Override
    public void onLoad() {
        this.saveDefaultConfig();
}
```

**Comment by MrHoller** [comment]
> i think it's not a bug, you just have two plugins with the same packgate names, next time name them like this for example: `me.author name.PluginName` `me.indian.Ostag` You just need to have different packgate names :)

I attached plugins with different names, if you take a clean server and run only these two plugins, you will get the same result

**Comment by IndianBartonka** [comment]
> > i think it's not a bug, you just have two plugins with the same packgate names, next time name them like this for example: `me.author name.PluginName` `me.indian.Ostag` You just need to have different packgate names :)
> 
> I attached plugins with different names, if you take a clean server and run only these two plugins, you will get the same result

yes the names are different but the package names are still the same which causes this error, it doesn't work the same as on bukkit

**Comment by IndianBartonka** [comment]
in `JukeboxMc`(https://github.com/LucGamesYT/JukeboxMC) I get a similar error, but there it just loads only one plugin

**Comment by MrHoller** [comment]
> Okay, I did make a mistake while copying. Now I fixed it and, on your advice, added it to onLoad() but the problem did not disappear
[sources_and_jars2.zip](https://github.com/PowerNukkitX/PowerNukkitX/files/11548626/sources_and_jars2.zip)

**Comment by IndianBartonka** [comment]
> > > > i think it's not a bug, you just have two plugins with the same packgate names, next time name them like this for example: `me.author name.PluginName` `me.indian.Ostag` You just need to have different packgate names :)
> > > 
> > > 
> > > I attached plugins with different names, if you take a clean server and run only these two plugins, you will get the same result
> > 
> > 
> > yes the names are different but the package names are still the same which causes this error, it doesn't work the same as on bukkit
> 
> Okay, I did make a mistake while copying. Now I fixed it and, on your advice, added it to onLoad() but the problem did not disappear [sources_and_jars2.zip](https://github.com/PowerNukkitX/PowerNukkitX/files/11548626/sources_and_jars2.zip)

Emmmm show you're console
